### DISCUSSION TITLE
Renovate changing package-lock.json differently than locally

### DISCUSSION THREAD

**Discussion by mcn18** [renovatebot/renovate#15241] [Request Help] [answered]
Renovate is currently updating one NPM package on my Gitlab repo and after the commit, the tests fail because it's missing dependencies. Here is what the commit looks like:
<img width="570" alt="Screen Shot 2022-04-21 at 11 01 48 PM" src="https://user-images.githubusercontent.com/5984571/164588118-b9bbcabc-ce59-4205-9089-3c5423886510.png">

When I manually change the package.json version and run npm install on my local computer, the package-lock.json is edited differently. 
`git diff --numstat
25      442     package-lock.json
1       1       package.json`
And the build works fine locally. Is there something I'm missing? I have tried RENOVATE_LOCK_FILE_MAINTENANCE as well as RENOVATE_REUSE_PACKAGE_LOCK and I can't see to get it to behave the same as locally. Both locally and renovate are using npm version 8.5.4 but it was behaving the same way on 8.7. 

Edit: I also just tested something else locally. When I `git checkout renovate/tailwindcss-3.x` and run `npm i` locally, the package-lock.json updates again and build works.

**Comment by mcn18** [comment, marked as answer]
Update: Rolling back to 32.26.3 as mentioned [here](https://github.com/renovatebot/renovate/issues/15211#issuecomment-1104977315) fixed the issue.

**Reply by viceice** [reply]
A fix is already merged to main but due to test failures on windows it's not yet released. I'm wokring on it.

**Reply by viceice** [reply]
PR on the way
- https://github.com/renovatebot/renovate/pull/15242
